howto/README.md

```diff
@@ -0,0 +1,49 @@
+# How do I ...
+
+## ... start a new Rails app?
+
+Use [suspenders]:
+
+```ruby
+$ gem install suspenders
+$ suspenders the-name-of-your-project-here
+$ cd the-name-of-your-project-here/
+$ rake
+```
+
+[suspenders]: https://github.com/thoughtbot/suspenders
+
+## ... feature-test a Rails app's Javascript?
+
+Use [capybara-webkit]. In your `Gemfile`:
+
+```ruby
+gem "capybara-webkit"
+```
+
+In `spec/support/capybara_webkit.rb` (for Rspec):
+
+```ruby
+Capybara.javascript_driver = :webkit
+
+Capybara::Webkit.configure do |config|
+  config.block_unknown_urls
+end
+```
+
+When writing a spec, you must set the `:js` flag for that test to make use of
+capybara-webkit. For example, in `spec/features/user_signs_in_spec.rb`:
+
+```ruby
+feature "Authentication", js: true do
```

Comment:
Since symbols are now treated like metadata keys with true values by default (https://github.com/rspec/rspec-core/pull/1006), should this be `feature "Authentication", :js do`?